README.md

```diff
@@ -239,6 +240,92 @@ This already goes with how CSP is currently enforcing its rules canonically in t
 
 `X-frames` and `frame-src` allow controlling what is allowed to be loaded into an `iframe`. Regardless of their use to deny loading content in iframes, ways to crete new same origin realms remain available, such as creating an `about:blank` iframe, opening a new tab using `open` API and more.
 
+## Consideration
+
+Some important points to consider
```

Comment:
You can probably drop this sentence

Reply:
For personal styling reasons, I highly prefer it when there's some plain content between 2 headers in markdown, so I changed it to be more indicative and self justifying instead (hope that's ok!) bff2e15b560299110d936035112bc291d4829873